lib/blockchain.go

```diff
@@ -2306,14 +2302,8 @@ func (bc *Blockchain) ProcessBlock(desoBlock *MsgDeSoBlock, verifySignatures boo
 	// accepted the block
 
 	// Signal the server that we've accepted this block in some way.
-	if bc.server != nil {
-		go func() {
-			bc.server.incomingMessages <- &ServerMessage{
-				Msg: &MsgDeSoBlockAccepted{
-					block: desoBlock,
-				},
-			}
-		}()
+	if bc.eventManager != nil {
+		bc.eventManager.blockAccepted(&BlockEvent{Block: desoBlock})
```

Comment:
I think this will cause the ProcessBlock to hang and/or run slower because it's not running it in a goroutine.